taigit/src/libraries/Taiga.ts

```diff
@@ -498,4 +498,29 @@ taiga_issues(projId : number) : Promise<Object>{
         output.push(new_entry);
     }
     return output;
-}
\ No newline at end of file
+}
+
+/**
+ * @summary This call return sprint velocity (unit points) based on sprint Id
+ * @param sprintId the ID for the access of sprint stats
+ * @returns
+ * * {
+ *          velocity : number //sprint velocity
+ * }
+ */
+export async function
+sprint_velocity_pts(sprintId : number) : Promise<Object> {
```

Comment:
@returns should be a number not an object

return type should be Promise<number>